services/settings/pkg/service/v0/servicedecorator.go

```diff
@@ -0,0 +1,83 @@
+package svc
+
+import (
+	"context"
+
+	settingsmsg "github.com/owncloud/ocis/v2/protogen/gen/ocis/messages/settings/v0"
+	settingssvc "github.com/owncloud/ocis/v2/protogen/gen/ocis/services/settings/v0"
+	"github.com/owncloud/ocis/v2/services/settings/pkg/settings"
+
+	"github.com/owncloud/ocis/v2/services/settings/pkg/config"
+	"github.com/owncloud/ocis/v2/services/settings/pkg/store/defaults"
+)
+
+// NewDefaultLanguageService returns a default language decorator for ServiceHandler.
+func NewDefaultLanguageService(cfg *config.Config, serviceHandler settings.ServiceHandler) settings.ServiceHandler {
+	defaultLanguage := cfg.DefaultLanguage
+	if defaultLanguage == "" {
+		defaultLanguage = "en"
+	}
+	return &defaultLanguageDecorator{defaultLanguage: defaultLanguage, ServiceHandler: serviceHandler}
+}
+
+type defaultLanguageDecorator struct {
+	defaultLanguage string
+	settings.ServiceHandler
+}
+
+// GetValueByUniqueIdentifiers implements the ValueService interface
+func (s *defaultLanguageDecorator) GetValueByUniqueIdentifiers(ctx context.Context, req *settingssvc.GetValueByUniqueIdentifiersRequest, res *settingssvc.GetValueResponse) error {
+	err := s.ServiceHandler.GetValueByUniqueIdentifiers(ctx, req, res)
+	if err != nil {
+		return err
+	}
+	if res.Value == nil {
+		res.Value = s.withDefaultLanguageSetting(req.AccountUuid)
+	}
```

Comment:
Shouldn't we only add the language if it was actually requested? I mean `GetValueByUniqueIdentifier` sounds to me like I can also get other settings except language here. But if request another value and it is not found, I would still get the default language setting...